core/src/ops/matmul/logic.rs

```diff
@@ -20,6 +20,16 @@ fn eval(
             return eval_t(a, b, a_trans, b_trans, c_trans, q_params, &|m, k, n| {
                 MMMWrapper::Quant((tract_linalg::ops().qmmm_i8_i32)(m, k, n))
             });
+        } else if (a.datum_type(), b.datum_type()) == (i8::datum_type(), i8::datum_type()) {
```

Comment:
Something is wrong. This condition is the same as the previous condition at line 19.

Reply:
Thanks for spotting that.